projects/core/src/i18n/numeric.pipe.ts

```diff
@@ -0,0 +1,44 @@
+import { DecimalPipe, getLocaleId } from '@angular/common';
+import { isDevMode, Pipe, PipeTransform } from '@angular/core';
+import { LanguageService } from '../site-context/facade/language.service';
+
+@Pipe({ name: 'cxNumeric' })
+export class CxNumericPipe extends DecimalPipe implements PipeTransform {
+  constructor(protected language: LanguageService) {
+    super(null);
```

Comment:
in `strict` mode, it will complain "Argument of type 'null' is not assignable to parameter of type 'string'."

Reply:
Picked

Reply:
@ChristophHi can we use "undefined" or  '' instead?

Reply:
fixed